exec/cnex/lib/net.c

```diff
@@ -249,7 +251,9 @@ void net_socket_recv(TExecutor *exec)
     int r = recv(s, (char*)buf, n, 0);
     if (r < 0) {
         perror("recv");
+        // Should we push a FALSE on the stack here?
```

Comment:
What does this comment mean? You either need to or you don't. In this case, you don't.

Reply:
Removed comment.